test/plugins/features/movies/index.test.js

```diff
@@ -18,6 +18,17 @@ describe('movies integration', () => {
       });
     });
 
+    it('lists movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        query: {}
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.data).to.have.lengthOf(4);
```

Comment:
Where do the 4 models come from? I don't see them inserted in the context of this test file.